utils/kafka/config.go

```diff
@@ -67,3 +67,18 @@ func WithErrorHandler(errorHandler func(error)) Option {
 		errorHandler: errorHandler,
 	}
 }
+
+type versionOption struct {
+	version sarama.KafkaVersion
+}
+
+func (opt *versionOption) apply(cfg *config) {
+	cfg.saramaConfig.Version = opt.version
+}
+
+// WithVersion specified the kafka cluster version to connect to
+func WithVersion(version sarama.KafkaVersion) Option {
+	return &versionOption{
+		version: version,
```

Comment:
@achichen should we consider to upgrade Orion sarama to 1.30.0?

Reply:
@praveensanap I was thinking the same. But the main reason to update sarama is to support newer kafka cluster, it has nothing to do with this change (this change should work with sarama v1.x). 

We can still do it but want to make clear it's not necessary.